src/types/index.d.ts

```diff
@@ -0,0 +1 @@
+export type HexString = string;
```

Comment:
> A couple of the the test helpers want to return a HexString. This is a type alias of string. I've redefined that alias here. I'm not really sure how that's going to play in other repos. If it conflicts, it's probably best to just have these functions return string.

TypeScript will handle this as a string. A type alias is just for better naming. When type checking there is no difference from the type `string`.